Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/PHP8/ClassWithLazyDependencies.php

```diff
@@ -1,5 +1,5 @@
-<?php
-namespace Neos\Flow\Tests\Functional\ObjectManagement\Fixtures;
+<?php /** @noinspection PhpMissingFieldTypeInspection */
+namespace Neos\Flow\Tests\Functional\ObjectManagement\Fixtures\PHP8;
```

Comment:
Should we maybe keep the original test for untyped lazy loading?

Reply:
there is still `AnotherClassWithLazyDependencies` (which was a copy from the original `ClassWithLazyDependencies`)

https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/AnotherClassWithLazyDependencies.php